survey/tests.py

```diff
@@ -19,4 +21,44 @@ def test_create_mode(self):
 
 class LegTests(TestCase):
 
-    pass
+    fixtures = ['modes.json']
+
+    def setUp(self):
+        self.survey_recipe = Recipe(models.Commutersurvey)
+        self.all_modes = models.Mode.objects.all()
+        self.leg_recipe = Recipe(
+            models.Leg,
+            checkin = self.survey_recipe.make(),
+            mode = cycle(self.all_modes)
+        )
+
+    def decimal_place_index(self, number):
```

Comment:
if this function represents the number of decimal places in the given number, can you rename it to "decimal_places" ?